easybuild/tools/configobj.py

```diff
@@ -1506,14 +1464,13 @@ def _decode(self, infile, encoding):
             # NOTE: Could raise a ``UnicodeDecodeError``
             return infile.decode(encoding).splitlines(True)
         for i, line in enumerate(infile):
-            if not isinstance(line, unicode):
+            if isinstance(line, str):
```

Comment:
Hmm, that's not fully equivalent, is it?

Especially not in Python 3, where `str` is also used for Unicode?

Reply:
Not really. But the code works on Python2 only anyway (no str.decode on Python 3) and https://github.com/easybuilders/easybuild-framework/pull/3282/files#diff-70567741de31395daaee88557c7c390dL1521 does it this way. This is the only place that uses `unicode`

As mentioned in Slack this code seems to be unused and has no tests or they would fail for Python3, so fixing those should be done later (or removed)

Reply:
Still, this change annoys me...

Why was it actually done, do you know?
What's wrong with the original `not isinstance(line, unicode)`?

Reply:
It was done manually because flake8 complained about the unknown name. IMO it is right to change it as it is consistent within the file (Line 1521 uses the same [new] check)

Alternative: Supress with `//noqa`

Reply:
OK